pkg/storage/config/schema_config.go

```diff
@@ -395,9 +395,9 @@ func (cfg *PeriodicTableConfig) UnmarshalYAML(unmarshal func(interface{}) error)
 // MarshalYAML implements the yaml.Marshaler interface.
 func (cfg PeriodicTableConfig) MarshalYAML() (interface{}, error) {
 	g := &struct {
-		Prefix string         `yaml:"prefix"`
-		Period model.Duration `yaml:"period"`
-		Tags   Tags           `yaml:"tags"`
+		Prefix string
+		Period model.Duration
+		Tags   Tags
 	}{
```

Comment:
it will produce yaml with properties starting from capital letters...

Reply:
Addressed in https://github.com/grafana/loki/pull/7981/commits/f585d81c3a6bdce2e58846c85d60c39ea5e24908